src/gui/socketapi.cpp

```diff
@@ -58,10 +66,80 @@
 #include <CoreFoundation/CoreFoundation.h>
 #endif
 
+
 // This is the version that is returned when the client asks for the VERSION.
 // The first number should be changed if there is an incompatible change that breaks old clients.
 // The second number should be changed when there are new features.
 #define MIRALL_SOCKET_API_VERSION "1.1"
+#define DEBUG qDebug() << "SocketApi: "
```

Comment:
Don't do that.  Use a debug category.  (As it allow to turn the log on and off with a env variable.

But I think you shoud cleanup most debug output.